### PR TITLE
Fixes #298 - Configuration of app's logger

### LINKED ISSUE #298: Log app events to file
**Is your feature request related to a problem? Please describe.**
App events are logged to stdout , which opens a separate console window when running app from OS shell.

**Describe the solution you'd like**
Basically command line window should not be open

- [ ] Add methods to configure logger ouput desination (see logrus docs)
- [ ] Choose stdout by default
- [ ] Wallet app GUI code should set log output to be sent to file 
- [ ] Add log file path in `QSettings`

**Describe alternatives you've considered**
Hard-coded log file path may lead to conflicts on different platforms . App settings directory could have been chosen as well .

### PR DESCRIPTION
**Pull request by AntiD2ta**
Fixes #298 

**Changes:**
- Add pipeline for change the level of logs from Settings view
- Add pipeline for change the output of logs from Settings view

**Does this change need to mentioned in CHANGELOG.md?**
yes | no

**Requires testing**
yes

**Comments about testing , should you have some**

### CODE REVIEW

File: src/models/config.go [hunk 1]
```diff
@@ -129,24 +130,42 @@ func (cs *ConfigSection) getOption(name string, path []string) *KeyValueStorage
 func (cs *ConfigSection) saveOptionValue(opt string, path []string, name string, value string) {
 	optV := cs.getOption(opt, path)
 	if optV == nil {
-		//log error
+		//TODO: log error
```
Comment (olemis) [inline src/models/config.go]: log error

File: src/models/config.go [hunk 2]
```diff
@@ -129,24 +130,42 @@ func (cs *ConfigSection) getOption(name string, path []string) *KeyValueStorage
...
 		return
 	}
 
 	optV.setValue(name, value)
 	data, err := json.Marshal(optV.keyValues)
 	if err != nil {
-		//log error
+		//TODO: log error
```
Comment (olemis) [inline src/models/config.go]: log error

File: src/models/config.go [hunk 3]
```diff
@@ -129,24 +130,42 @@ func (cs *ConfigSection) getOption(name string, path []string) *KeyValueStorage
...
 		return
 	}
 	cs.sm.Save(opt, path, string(data))
 
+	if opt == "log" {
+		if name == "level" {
+			level, err := logging.LevelFromString(value)
+			if err != nil {
+				//TODO: log error
```
Comment (olemis) [inline src/models/config.go]: log error

File: src/models/config.go [hunk 4]
```diff
@@ -129,24 +130,42 @@ func (cs *ConfigSection) getOption(name string, path []string) *KeyValueStorage
...
+			} else {
+				logging.SetLevel(level)
+			}
+		} else if name == "output" {
+			writer, err := logging.GetOutputWriter(value)
+			if err != nil {
+				//TODO: log error
```
Comment (olemis) [inline src/models/config.go]: log error

File: src/models/config.go [hunk 5]
```diff
@@ -129,24 +130,42 @@ func (cs *ConfigSection) getOption(name string, path []string) *KeyValueStorage
...
+			}
+		}
+	}
 }
 
 func (cs *ConfigSection) getValue(opt string, path []string, name string) string {
 	optV := cs.getOption(opt, path)
 	if optV == nil {
-		//log error
+		//TODO: log error
```
Comment (olemis) [inline src/models/config.go]: log error

File: src/ui/Settings.qml [hunk 6]
```diff
@@ -288,6 +305,65 @@ Page {
...
+                    selectByMouse: true
+
+                    placeholderText: qsTr("Log output")
+                    onTextChanged: {
+                        updateFooterButtonsStatus();
+                    }
+                }
+            }
+        }
+        // RowLayout {
```
Comment (olemis) [inline src/ui/Settings.qml]: remove if not needed

**Review by olemis — CHANGES_REQUESTED** [review]
@stdevAntiD2ta more tests need be written

**Comment by olemis** [comment]
> @stdevAntiD2ta more tests need be written

for coverage to be reported ok

File: src/ui/Settings.qml (hunk 6 above)
Comment (AntiD2ta) [reply src/ui/Settings.qml]: That code is for the initial Combo Box that I made, I left it commented for @stdevCrow to fix it

Comment (AntiD2ta) [reply src/ui/Settings.qml]: Code removed

**Review by olemis — APPROVED** [review]

**Pull request merged**